### PR TITLE
Feature/remove content filter

### PR DESCRIPTION
**Pull request by allandegnan**
### What

find-a-dataset should only work with 

### How to review

Checkboxes missing, but `filter=dataset_landing_page` sent to api.

### Who can review

!me

### CODE REVIEW

**Comment by lindenmckenzie** [comment]
@allandegnan has this PR been superseded or is it still relevant?

**Comment by allandegnan** [comment]
> @allandegnan has this PR been superseded or is it still relevant?

This is the end PR, but needs a fixup and requires this: https://github.com/ONSdigital/dp-search-api/pull/200

File: config/config.go [hunk 1]
```diff
@@ -72,9 +72,9 @@ func get() (*Config, error) {
 		DefaultOffset:                           0,
 		DefaultPage:                             1,
 		DefaultSort:                             "relevance",
-		EnableCensusTopicFilterOption:           false,
-		EnableCensusPopulationTypesFilterOption: false,
-		EnableCensusDimensionsFilterOption:      false,
+		EnableCensusTopicFilterOption:           true,
+		EnableCensusPopulationTypesFilterOption: true,
+		EnableCensusDimensionsFilterOption:      true,
```
Comment (lindenmckenzie) [inline config/config.go:77]: Don't think these should be set to true by default yet.

**Review by lindenmckenzie — COMMENTED** [review]
Other than the config bools, this looks fine.

File: config/config.go (hunk 1 above)
Comment (nshumoogum) [reply config/config.go]: I was about to make the same comment, these should be false and then environment secrets containing these flags should be updated

File: features/dataset_finder.feature
Comment (lindenmckenzie) [inline features/dataset_finder.feature:1]: Accepted deletion of test as this feature is not here to retro-fit mocks to everything in search controller. Task to be picked up as tech debt.

**Review by lindenmckenzie — APPROVED** [review]

**Pull request merged**